### PR TITLE
CORTX-30691 - Community Build instructions improvements

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
### Describe your changes in brief

Improved community build instructions. 

- Added step to install Git Package
- Added minimum RAM and CPU requirements.

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

-----
[View rendered doc/community-build/docker/cortx-all/README.md](https://github.com/shailesh-vaidya/cortx/blob/CORTX-30691-Community-Build/doc/community-build/docker/cortx-all/README.md)

### CODE REVIEW

File: doc/community-build/docker/cortx-all/README.md [hunk 1]
```diff
@@ -46,11 +57,11 @@ This document provides step-by-step instructions to build CORTX binaries and con
    docker run --rm -v /var/artifacts:/var/artifacts -v /mnt/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:rockylinux-8.4 make clean cortx-all-rockylinux-image
    ```
    
-   **Note:** This process takes some time to complete building the CORTX packages during `/var/artifacts/0 /` implementation phase.
+   **Note:** This process takes around 5Hrs to build entire CORTX packages with minimum VM specifications. This time will vary based on VM specification and Network Speed.
```
Comment (mukul-seagate11) [inline doc/community-build/docker/cortx-all/README.md:60]: rephrase as,

**Note:** This process takes around 5Hrs to build entire CORTX packages with minimum VM specifications where time can vary based on VM specification and network speed.

File: doc/community-build/docker/cortx-all/README.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
 This document provides step-by-step instructions to build CORTX binaries and container images.
 
 ## Prerequisites
+- Please create Linux VM with CentOS 7.9 or Rocky Linux 8.4 with below minium requirements.
+    - RAM: 8GB
+    - Processor: 4
+
+- Please ensure following minimum disk space is available. 
+    - 70GB : Docker Home Directroy. Default docker home directory is /var/lib/docker. You can modify it based on disk layout.
```
Comment (mukul-seagate11) [inline doc/community-build/docker/cortx-all/README.md:11]: rephrase as,

- Ensure following minimum disk space are available:
    - 70GB : Docker Home Directory where default docker home directory is /var/lib/docker. You can modify the file based on disk layout.
    - 30GB : /mnt . Use this directory as part of build process. You can use `/mnt` from system disk if you have enough disk space available.

File: doc/community-build/docker/cortx-all/README.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
 This document provides step-by-step instructions to build CORTX binaries and container images.
 
 ## Prerequisites
+- Please create Linux VM with CentOS 7.9 or Rocky Linux 8.4 with below minium requirements.
```
Comment (mukul-seagate11) [inline doc/community-build/docker/cortx-all/README.md:6]: rephrase as,

Create Linux VM with CentOS 7.9 or Rocky Linux 8.4 with below minimum requirements:

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/community-build/docker/cortx-all/README.md (hunk 1 above)
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md]: Fixed please review

File: doc/community-build/docker/cortx-all/README.md (hunk 2 above)
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md]: Fixed please review

File: doc/community-build/docker/cortx-all/README.md (hunk 3 above)
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md]: Fixed please review

**Review by mukul-seagate11 — APPROVED** [review]
looks good, hence merging

**Pull request merged**